src/index.ts

```diff
@@ -415,9 +415,31 @@ export async function install(
     chunkFileNames: 'common/[name]-[hash].js',
   };
   if (Object.keys(depObject).length > 0) {
-    const packageBundle = await rollup.rollup(inputOptions);
-    await packageBundle.write(outputOptions);
+    // Main Snowpack Install 🚀
+    try {
+      const packageBundle = await rollup.rollup(inputOptions);
+      await packageBundle.write(outputOptions);
+    } catch (err) {
+      // Main Snowpack Install error handling
+      const {loc} = err as rollup.RollupError; // Note: err contains a ton of useful information for error handling
+
+      // if there’s a suggestion based on the file extension, show it
+      const invalidFile = loc.file || ''; // Rollup erred on this exact file
+      const suggestion = fileExtSuggestions[path.extname(invalidFile)];
+      if (suggestion) {
+        const relativeName = invalidFile.replace(cwd, '').replace(path.sep, '/'); // use forward slashes for display
```

Comment:
Tell me if this is dumb for the Windows test: it shows only forward slashes for display. I figured this is fine, since we’re using forward-slash ESM world.

While I never want tests to influence the code, I don’t think I want to have, say, an `expected-output-win.txt`. Nor do I want to just convert every `\` to `/` in the output comparison indiscriminately.

Since this error isn’t encountered often, and doesn’t affect much, I think a tiny change here is fine because it keeps the tests pretty strict.